src/cunumeric/matrix/solve.cc

```diff
@@ -0,0 +1,100 @@
+/* Copyright 2021-2022 NVIDIA Corporation
```

Comment:
a nitpick: this file (and all the other new files in this PR) didn't exist in 2021. please update the copyright year accordingly.

Reply:
changed to 2022 only.